frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.jsx

```diff
@@ -145,17 +164,32 @@ export default class Smart extends Component {
       ? color("error")
       : color("success");
 
-    const changeDisplay = (
-      <span style={{ fontWeight: 900 }}>
-        {formatNumber(Math.abs(lastChange), { number_style: "percent" })}
-      </span>
-    );
-    const separator = (
-      <PreviousValueSeparator gridSize={gridSize}>•</PreviousValueSeparator>
-    );
-    const granularityDisplay = (
-      <span style={{ marginLeft: 5 }}>{jt`last ${granularity}`}</span>
+    const titleLinesCount = getTitleLinesCount(height);
+
+    const tooltipSeparator = <Separator>•</Separator>;
+
+    const changeDisplay = formatChangeAutoPrecision(lastChange, {
+      fontFamily,
+      fontWeight: 900,
+      width: getChangeWidth(width),
+    });
+
+    const granularityDisplay = jt`last ${granularity}`;
+    const previousValueDisplay = formatValue(
+      previousValue,
+      settings.column(column),
     );
+    const disabledSeparator = ""; // avoid creating new translation key
+    const previousValueContent = jt`${disabledSeparator} was ${previousValueDisplay} ${granularityDisplay}`;
```

Comment:
I'm not sure why we couldn't do the same as `tooltipSeparator` e.g.

```suggestion
    const previousSeparator = (
      <PreviousValueSeparator>•</PreviousValueSeparator>
    );
    const previousValueContent = jt`${previousSeparator} was ${previousValueDisplay} ${granularityDisplay}`;
```

This makes sure the value we use for both tooltip and the value (non-tooltip) uses a similar structure.

Reply:
> why we couldn't do the same as `tooltipSeparator`

Tooltip will always show the separator.
In the card, separator will not be shown if this this text appears in a separate line, which depends on CSS breakpoints. 
Because it depends on CSS breakpoints, we can't use the same styled component in the card and in the tooltip.

Reply:
Alright! Things are much simpler now: 84ae77da84eed1b125f48002e5a788404d673f68